packages/app/android/src/reactnative/java/io/invertase/firebase/common/ReactNativeFirebaseJSON.java

```diff
@@ -68,6 +68,26 @@ public String getStringValue(String key, String defaultValue) {
     return jsonObject.optString(key, defaultValue);
   }
 
+  public ArrayList<String> getArrayValue(String key) {
+    if (jsonObject == null) return new ArrayList<String>();
+
+    try {
+      ArrayList<String> result = new ArrayList<String>();
+      
+      JSONArray array = jsonObject.optJSONArray(key);
+      if(array != null){
+        for (int i=0;i<array.length();i++) {
+          result.add(array.getString(i));
+        }
+      }
+
+      return result;
+    }
+    catch (JSONException e) {
+      return new ArrayList<String>();
+    }
```

Comment:
suggest single result init, nit spacing
```suggestion
    ArrayList<String> result = new ArrayList<String>();
    if (jsonObject == null) return result;

    try {      
      JSONArray array = jsonObject.optJSONArray(key);
      if (array != null) {
        for (int i = 0; i < array.length(); i++) {
          result.add(array.getString(i));
        }
      }
    }
    catch (JSONException e) {
      // do nothing
    }
    
    return result;
```